vignettes/openxlsx2_charts_manual.Rmd

```diff
@@ -115,7 +118,7 @@ wb$add_worksheet("add_mschart")$add_mschart(dims = "A10:G25", graph = mylc)
 # write data starting at B2
 wb$add_worksheet("add_mschart - wb_data")$
   add_data(x = mtcars, dims = "B2")$
-  add_data(x = data.frame(name = rownames(mtcars)), dims = "A2")
+  add_data(x = data.frame(name = rownames(mtcars)), dims = wb_dims(start_row = 2))
```

Comment:
I don't really like this. It's an interesting solution you came up with, but this is still strange :D

Reply:
I will revert in this case :) Not much added-value indeed